.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)